cdap-docs/admin-manual/source/upgrading/ambari.rst

```diff
@@ -0,0 +1,29 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2016 Cask Data, Inc.
+
+.. _admin-upgrading-ambari:
+
+==================================
+Upgrading CDAP using Apache Ambari
+==================================
+
+
+Upgrading CDAP
+==============
+Currently, CDAP **cannot** be upgraded by using Apache Ambari. 
+
+To upgrade CDAP installations that were installed and are managed with Apache Ambari, please
+follow our instructions for upgrading CDAP installations that were installed with a
+Package Manager, either RPM or Debian:
```

Comment:
... "either Yum or Apt:".  Nitpick: it should probably just be "installed using packages" since it's not necessary to use the managers (ie download our rpm-bundle and then "rpm -ivh cdap*.rpm")